_posts/2021-06-01-updates.md

```diff
@@ -0,0 +1,59 @@
+
+
+
+### New Stats API
+
+We have also released a new set of APIs centered around statistics retrieval. There are different REST endpoints to retrieve [a single statistical value](https://docs.datacommons.org/api/rest/stat_value.html), [a statistical time series](https://docs.datacommons.org/api/rest/stat_series.html) or [the entire collection of statistical data for a set of places](https://docs.datacommons.org/api/rest/stat_all.html). We have used these APIs to build the new Scatter Plot Explorer and hope this enables other applications too.
+
+
+
+### New Data
+
+As always, we continue to add more data to the Data Commons Graph. Some recent additions include:
+- Indian Census (e.g. [houseless](https://datacommons.org/browser/country/IND?statVar=Count_Household_Houseless_Rural) and [rural literacy](https://datacommons.org/browser/wikidataId/Q15116?statVar=Count_Person_Literate_Rural) populations)
+- Reserve Bank of India's Poverty data (e.g. [Rural population below poverty in Andhra Pradesh](https://datacommons.org/browser/wikidataId/Q1159?statVar=Count_Person_Rural_BelowPovertyLevelInThePast12Months))
+- FDA and additional drug information (e.g. [drug](https://datacommons.org/browser/bio/CHEMBL512) from FDA, ChEMBL, PharmGKB, etc.)
+- Improved Covid-19 statistics (e.g. [vaccination stats from ourworldindata.org](https://datacommons.org/browser/country/IND?statVar=CumulativeCount_Vaccine_COVID_19_Administered))
+- US Energy Information Administration (e.g. [coal](https://autopush.datacommons.org/browser/country/USA?statVar=Quarterly_Consumption_Coal_ElectricPower) and [natural gas consumption](https://autopush.datacommons.org/browser/country/USA?statVar=Quarterly_Consumption_Fuel_ForElectricityGeneration_NaturalGas) for electricity)
+- Expanded international data from World Bank (e.g., [crime](https://autopush.datacommons.org/browser/country/CHN?statVar=Count_CriminalActivities_MurderAndNonNegligentManslaughter_AsFractionOf_Count_Person), [health](https://autopush.datacommons.org/browser/country/IND?statVar=Count_Person_15OrMoreYears_Smoking_AsFractionOf_Count_Person_15OrMoreYears)[stats](https://autopush.datacommons.org/browser/country/IND?statVar=Count_Person_Upto4Years_Overweight_AsFractionOf_Count_Person_Upto4Years))
+- Updated data from existing sources, including:
+  - FBI Crime (till [2019](https://datacommons.org/browser/geoId/06?statVar=Count_CriminalActivities_ViolentCrime))
+  - BLS Unemployment (till [2021 Q1](https://datacommons.org/browser/geoId/06?statVar=UnemploymentRate_Person))
+  - World Bank (till [2019](https://www.google.com/search?q=population+of+china&oq=population+of+china))
+
+
+
+As always, we are eager to hear from you! Please [share your feedback](https://datacommons.org/feedback) with our team.
+
+
+
+Carolyn on behalf of the Data Commons team
```

Comment:
nit: add period

Reply:
skipping this since it's the signature line and not really a sentence